src/components/ContactGroups/ContactGroupManager.jsx

```diff
@@ -1,46 +1,62 @@
-import React from 'react'
+import React, { Component } from 'react'
 import PropTypes from 'prop-types'
-import { Button } from 'cozy-ui/react'
+import { Button, Overlay } from 'cozy-ui/react'
 import { translate } from 'cozy-ui/react/I18n'
-import SelectBox, {
-  reactSelectControl,
-  CheckboxOption
-} from 'cozy-ui/react/SelectBox'
+
+import classNames from 'classnames'
+
+import SelectBox, { CheckboxOption, components } from 'cozy-ui/react/SelectBox'
 import Icon from 'cozy-ui/react/Icon'
 import palette from 'cozy-ui/stylus/settings/palette.json'
 
 import IconGroups from '../../assets/icons/groups.svg'
 import IconDown from '../../assets/icons/down.svg'
+import ContactGroupCreation from './ContactGroupCreation'
 
-const MainButton = ({ t }) => (
-  <Button
-    theme="secondary"
-    size="small"
-    icon={<Icon icon={IconGroups} color={palette['coolGrey']} />}
-    label={t('groups.manage')}
-  >
-    <Icon
-      icon={IconDown}
-      color={palette['coolGrey']}
-      width="12"
-      className="group-manager__indicator"
-    />
-  </Button>
-)
+const MainButton = ({ t, selectProps: { toggleMenuIsOpen } }) => {
+  return (
+    <Button
+      theme="secondary"
+      size="small"
+      icon={<Icon icon={IconGroups} color={palette['coolGrey']} />}
+      label={t('groups.manage')}
+      onClick={() => {
+        toggleMenuIsOpen()
+      }}
+    >
+      <Icon
+        icon={IconDown}
+        color={palette['coolGrey']}
+        width="12"
+        className="group-manager__indicator"
+      />
+    </Button>
+  )
+}
 
 MainButton.propTypes = {
   t: PropTypes.func.isRequired
 }
 
 const MainButtonWithTranslation = translate()(MainButton)
 
-const MainButtonControl = reactSelectControl(<MainButtonWithTranslation />)
-
+const MainButtonControl = props => <MainButtonWithTranslation {...props} />
 const customStyles = {
   container: base => ({
     ...base,
     display: 'inline-block',
-    verticalAlign: 'middle'
+    verticalAlign: 'middle',
+    zIndex: 100000
+  }),
+  menu: base => ({
+    ...base
+  }),
+  noOptionsMessage: () => ({}),
```

Comment:
Why do we need to reset the styles of this component?

Reply:
Because I wrote all the needed style here: https://github.com/cozy/cozy-contacts/pull/316/files#diff-0f0d1cddc45a5df3cf945d6f6311fab7R29 and didn't want to add !important everywhere ;)